src/NuGetGallery/Views/Packages/DisplayPackage.cshtml

```diff
@@ -504,6 +504,59 @@
                 }
             }
 
+            @if(Model.IsGitHubUsageEnabled)
+            {
+                <h2>
+                    <a role="button" data-toggle="collapse" data-target="#github-usage"
+                       aria-expanded="false" aria-controls="github-usage" id="show-github-usage">
+                        <i class="ms-Icon ms-Icon--ChevronRight" aria-hidden="true"></i>
+                        <span>GitHub Usage</span>
+                    </a>
+                </h2>
+            
+                <style>
+                    .btn-gh-repo {
```

Comment:
Can this CSS be moved to this file? https://github.com/NuGet/NuGetGallery/blob/master/src/Bootstrap/less/theme/page-display-package.less